examples/build_file_generation/WORKSPACE

```diff
@@ -23,11 +23,11 @@ http_archive(
 # to include https://github.com/bazelbuild/bazel-gazelle/commit/2834ea4
 http_archive(
     name = "bazel_gazelle",
-    sha256 = "0bb8056ab9ed4cbcab5b74348d8530c0e0b939987b0cfe36c1ab53d35a99e4de",
-    strip_prefix = "bazel-gazelle-2834ea44b3ec6371c924baaf28704730ec9d4559",
+    sha256 = "fd8d852ebcb770b41c1c933fc3085b4a23e1426a1af4e791d39b67bb8d894eb7",
+    strip_prefix = "bazel-gazelle-41b542f9b0fefe916a95ca5460458abf916f5fe5",
     urls = [
         # No release since March, and we need subsequent fixes
```

Comment:
I think this is no longer true?

Reply:
Subsequent releases didn't include the fix we needed yet. I believe Gazelle may release again soon, then we can get rid of this.